README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portfolio-mv
 
-> Built the header section of my portfolio using the figma files for the system design
+The goal was to be able to parse a Figma design to create a beautiful UI using Flexbox, Grid CSS, images and backgrounds, while sticking to the best practices.using this [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in figma. You can view the live site [here](https://edward-kabue.github.io/Portfolio-mv/).
 
 ![the design](./images/Header-section.png)
```

Comment:
- [ ] OPTIONAL: I recommend you adding a new section in README and add a preview link under the title, eg.: 

```
## Live demo
You can view the live site [here](https://edward-kabue.github.io/Portfolio-mv/)
```

Reply:
Ok will do so thank you